scripts/open_url_in_instance.sh

```diff
@@ -3,16 +3,13 @@
 # author: Thore Bödecker (foxxx0)
 
 _url="$1"
-_qb_version='0.10.1'
+_qb_version='1.0.3'
```

Comment:
1.0.4 was just released, so maybe we should bump this up to that?

https://github.com/qutebrowser/qutebrowser/releases/tag/v1.0.4